services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email or "",
+        first_name=first_name or "",
+        last_name=last_name or ""
+    )
```

Comment:
if you don't have some fields, you writing `""` in the field, but it is not empty field
if this fields = None, you don't need set it
for example -> see `update_user()`